integration/terraform/ec2/linux/variables.tf

```diff
@@ -0,0 +1,72 @@
+variable "ec2_instance_type" {
+  type = string
+  default = "t3a.xlarge"
+}
+
+variable "key_name" {
+  type = string
+  default = "cwagent-integ-test-key"
+}
+
+variable "iam_instance_profile" {
+  type = string
+  default = "CloudWatchAgentServerRole"
+}
+
+variable "vpc_security_group_ids" {
+  type = list(string)
+  default = ["sg-013585129c1f92bf0"]
+}
+
+variable "region" {
+  type = string
+  default = "us-west-2"
+}
+
+variable "aws_access_key" {
+  type = string
+}
+
+variable "aws_secret_access_key" {
+  type = string
+}
+
+variable "ami" {
+  type = string
+  default = ""
+}
+
+variable "ssh_key" {
+  type = string
+  default = ""
+}
+
+variable "user" {
+  type = string
+  default = ""
+}
+
+variable "github_sha" {
+  type = string
+  default = ""
+}
+
+variable "github_repo" {
+  type = string
+  default = ""
+}
+
+variable "package" {
```

Comment:
This looks really ambiguous as just `package` to me (made a point in another file about it already). Can we add a `description` value here to explain what this is used for?